Makefile.v8

```diff
@@ -22,21 +26,35 @@ plv8_config.h: $(AUTOV8_LIB)
 
 $(AUTOV8_DIR):
 	mkdir -p build
-	curl -L 'https://github.com/v8/v8/archive/$(AUTOV8_VERSION).tar.gz' \
-		| tar zxf - -C build
+	cd build; fetch v8; cd v8; git checkout $(AUTOV8_VERSION); gclient sync
+	mv build/v8 $(AUTOV8_DIR)
 
 $(AUTOV8_LIB): $(AUTOV8_DIR)
-	env CXXFLAGS=-fPIC $(MAKE) -C build/v8-$(AUTOV8_VERSION) dependencies
-	env CXXFLAGS=-fPIC $(MAKE) -C build/v8-$(AUTOV8_VERSION) native
+	env CXXFLAGS=-fPIC CFLAGS=-fPIC $(MAKE) -C build/v8-git-mirror-$(AUTOV8_VERSION) native -j12
```

Comment:
This -j12 is not good.  The j option should be given from the command line and be propagated, as it depends on the compile environment how many parallel jobs are acceptable.